dts/arm/st/f0/stm32f0.dtsi

```diff
@@ -49,8 +49,9 @@
 
 		clk_lse: clk-lse {
 			#clock-cells = <0>;
-			compatible = "fixed-clock";
+			compatible = "st,stm32f0-lse-clock";
```

Comment:
I don't see the point of adding series specific bindings if they are all the same.
Please use only one st,stm32-lse-clock binding when possible.

Reply:
done

Reply:
Besides, the rebase, which is coming soon. Do you already have a rough timeline when #44312 will be merged? Seems to be pretty ready?

Reply:
Rebase is done as well.